### DISCUSSION TITLE
Multipart monographs

### DISCUSSION THREAD

**Discussion by pan-zhuo** [uwlib-cams/sinopia_maps#107] [come back to it]
Multipart monographs can be described either separately or as a whole, often depending on whether individual titles have meaning.

A multipart monograph issued simultaneously as a whole may embody either a single work or a static aggregating work.

A multipart monograph issued over time as a whole can only embody a successive aggregating work (diachronic work).

Each individual unit may embody either a single work or a static aggregating work, regardless of the mode of issuance.

Problem
---
[LC-PCC Policy](https://www.loc.gov/aba/rda/mgd/mg-basicCatalogingDecisions.pdf) (page 3) seems to bend the rules in the case of multipart monographs issued over time by treating them as static works.

<img width="443" alt="image" src="https://user-images.githubusercontent.com/100716525/230799110-d2e7a766-caba-4ae9-a7c4-f2fec8a4973d.png">

I made a different version which I think represents the RDA modeling more accurately.

|Type of work|Mode of issuance|Extension plan|
|---|---|---|
|Monograph|Single unit|Static plan|
|Multipart monograph issued simultaneously|Multiple units|Static plan|
|Multipart monograph issued over time|Multiple units|Successive determinate|

Questions
---
Should the print monograph RTs include describing multipart monographs?
(The current RTs should be able to support this without overhaul, though guidance is lacking for choosing appropriate templates.)

Which model do we prefer for multipart monographs issued over time?
(For best RDA we may want to go with successive determinate. However, I'm not confident that I understand diachronic works well enough to provide readable guidance...)

**Comment by briesenberg07** [comment]
**Responding to only one of two questions:**

> Should the print monograph RTs include describing multipart monographs?  

*Let's complete templates and guidance for single-unit, static-plan monographs before attempting?*
